src/main/java/me/youhavetrouble/justchat/ConfigReload.java

```diff
@@ -0,0 +1,39 @@
+package me.youhavetrouble.justchat;
+
+import org.bukkit.plugin.Plugin;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigReload {
+    private static final Plugin plugin = JustChat.plugin;
+    private static final HashMap<Message, String> pluginMessages = new HashMap<>();
```

Comment:
Why is this here? We can just create a getter and setter for the enum itself, there's no need for this map.
```java
public enum ConfigValue {

    CHAT_FORMAT("default format or something");
    
    private Component message;
    ConfigValue(String message) {
      this.message = MiniMessage.miniMessage().deserialize(message);
    }

  public Component getMessage() {
    return message;
  }

}
```